src/navigation-plugin.tsx

```diff
@@ -212,6 +219,9 @@ export class NavigationPlugin extends KalturaPlayer.core.BasePlugin {
       if (this._getCuePointType(cue) === ItemTypes.Caption && this._itemsFilter[ItemTypes.Caption]) {
         captionData.push(prepareCuePoint(cue, ItemTypes.Caption, isLive));
       }
+      if (this._getCuePointType(cue) === ItemTypes.QuizQuestion && this._itemsFilter[ItemTypes.QuizQuestion]) {
```

Comment:
@semarche-kaltura this will be removed on next PR, as we discussed - the cuepoint preparation will be executed inside new callback method for listener to changes in quiz.